src/SFA.DAS.PR.Data/EntityConfiguration/AccountLegalEntityConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using SFA.DAS.PR.Domain.Entities;
+using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.PR.Data.EntityConfiguration;
+
+[ExcludeFromCodeCoverage]
+public class AccountLegalEntityConfiguration : IEntityTypeConfiguration<AccountLegalEntity>
+{
+    public void Configure(EntityTypeBuilder<AccountLegalEntity> builder)
+    {
+        builder
+            .HasKey(p => p.Id);
+
+        //builder
+        //    .HasQueryFilter(ale => ale.Deleted == null);
+
+        builder
```

Comment:
I thought all these foreign key configuration was not needed as it follows EF conventions, was it not working without these?